client/src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,27 @@
+import './styles.scss';
+import { navLinks } from '../../data/navBar';
+
+const NavBar = () => {
+  return (
+    <nav>
+      <ul className='container-xl'>
+        {navLinks.map(link => {
+          if(link.toggle) {
+            return (
+              <li>
+                <span>{link.title}</span>
+                <i class="fa-solid fa-angle-down"></i>
+              </li>
+            )
+          } else {
+            return (
+              <li>{link.title}</li>
+            )
+          }
```

Comment:
```suggestion
        {navLinks.map(({title, toggle}) => (
              <li>
                <span>{title}</span>
                {toggle && <i class="fa-solid fa-angle-down"></i>}
              </li>
            )
```
ngoài ra các biến boolean thì đặt theo kiểu isToggle, hasSubmenu,...